orttraining/orttraining/test/python/how_to_install_training_test_dependencies.md

```diff
@@ -1,15 +1,15 @@
 ## Installing ORT Test Dependencies Locally
 
-To install all development test dependencies for ONNX Runtime training, run the below two commands:
+To install all development test dependencies for ONNX Runtime training for ORTModule, run the below two commands:
 
 
-Due to a [bug on DeepSpeed](https://github.com/microsoft/DeepSpeed/issues/663), we need a two step process for installing dependencies ORTModule dependencies.
+Due to a [bug on DeepSpeed](https://github.com/microsoft/DeepSpeed/issues/663), we need a two step process for installing ORTModule dependencies.
 
 1. Install first set of dependencies:
 
    ```sh
-   pip install -r tools/ci_build/github/linux/docker/scripts/training/requirements.txt
+   pip install -r tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage1/requirements.txt
 
 2. Install second set of dependencies for ortmodule:
```

Comment:
clarify *why* a second set is needed

Reply:
# Due to a [bug on DeepSpeed](https://github.com/microsoft/DeepSpeed/issues/663), we install it separately through ortmodule/stage2/requirements.txt

Reply:
There is a comment on that earlier in this file that addresses this (line6).